artifacts/notification-as-cloud-event.yaml

```
@@ -0,0 +1,285 @@
+      summary: "Cloud Event notification endpoint to notify consumer that statement of fact had occurred"
+      description: |
+        INFORMATIVE ENDPOINT: The value of this endpoint is freely declared by each client app by means of resource-based
+        subscription or instance-based subscription. /your_webhook_notification_url is 
+        just a convention naming referring to an absolute URL, indeed the one indicated by API client
+        in the triggering of the procedure (resource-based or instance-based).  In this way, it represents an absolute
+        URL, i.e.: notifications won't be sent to /event-notification/vX/your_webhook_notification_url.
+      operationId: sendEvent
+      requestBody:
+        required: true
+        content:
+          application/cloudevents+json:
+            schema:
+              $ref: "#/components/schemas/CloudEvent"
+            examples:
+              QOS_STATUS_CHANGED_EXAMPLE:
+                $ref: '#/components/examples/QOS_STATUS_CHANGED_EXAMPLE'
+
+      responses:
+        201:
+          description: Created
+        202:
+          description: Accepted
+        204:
+          description: No Content
+        200:
+          description: Ok
+          content:
+            application/json:
+              schema:
+                type: object
+        400:
+          $ref: "#/components/responses/Generic400"
+        401:
+          $ref: "#/components/responses/Generic401"
+        403:
+          $ref: "#/components/responses/Generic403"
+        410:
+          $ref: "#/components/responses/Generic410"
+        415:
+          $ref: "#/components/responses/Generic415"
+        429:
+          $ref: "#/components/responses/Generic429"
+        500:
+          $ref: "#/components/responses/Generic500"
+        503:
+          $ref: "#/components/responses/Generic503"
+components:
+  securitySchemes:
+    oAuth2ClientCredentials:
+      type: oauth2
+      flows:
+        clientCredentials:
+          tokenUrl: '{tokenUrl}'
+          scopes: {}
+  schemas:
+    ErrorInfo:
+      type: object
+      required:
+        - status
+        - code
+        - message
+      properties:
+        status:
+          type: integer
+          description: HTTP response status code
+        code:
+          type: string
+          description: Code given to this error
+        message:
+          type: string
+          description: Detailed error description
+
+    DateTime:
+      type: string
+      format: date-time
+      description: |
+        Timestamp of when the occurrence happened. Must adhere to RFC 3339. 
+        WARN: This optional field in CloudEvents specification is required in CAMARA APIs implementation.
+      example: '2018-04-05T17:31:00Z'
+
+    Source:
+      type: string
+      format: uri-reference
+      minLength: 1
+      description: "Identifies the context in which an event happened(example: 'com.orange.camara.qod')"
```

URI-reference is not Reverse-DNS
```suggestion
      description: "Identifies the context in which an event happened - be a non-empty `URI-reference`.
```